src/token/token.setting.ts

```diff
@@ -1,12 +1,19 @@
-import { TEN_MINUTES } from '../data/constants.js';
+import { EXPIRATION_TIME } from '../data/constants.js';
 
 export class TokenSetting {
+  oneMinuteInMs: number;
+
+  constructor() {
+    this.oneMinuteInMs = 60000;
+  }
+
   saveToken(token: string): void {
     localStorage.setItem('token', token);
   }
 
   setExpireTime(): void {
-    const tokenExpireTime: number = Date.now() + TEN_MINUTES;
+    const tenMinutes = EXPIRATION_TIME * this.oneMinuteInMs;
+    const tokenExpireTime: number = Date.now() + tenMinutes;
```

Comment:
you can multiply ms and expiration time right here. it won't make the code more difficult to read